### ISSUE TITLE
PostgreSQL table with enum column and default value

### ISSUE THREAD

**Issue by foarsitter** [xnuinside/simple-ddl-parser#286]
Thank you for your beautiful piece of work! 

Parsing a table with an enum column with a default results in a missing table. Removing the default solves this problem. 

Here is a minimum viable DDL: 

```postgesql
CREATE TYPE public.blog_categories_visibility AS ENUM (
    'public',
    'protected',
    'private'
);

CREATE TABLE public.blog_categories (
    id integer NOT NULL,
    client_id integer,
    name character varying(255),
    slug character varying(255),
    color character varying(16),
    published boolean DEFAULT true NOT NULL,
    visibility public.blog_categories_visibility DEFAULT 'public'::public.blog_categories_visibility NOT NULL,
    filter_id integer
);
```

Thanks again!

**Comment by xnuinside** [comment]
@foarsitter thanks for opening the issue, will be fixed in version 1.6.2, working on it

**Comment by xnuinside** [comment]
@foarsitter released in version 1.7.0 https://github.com/xnuinside/simple-ddl-parser/pull/287 https://pypi.org/project/simple-ddl-parser/1.7.0/ . Thanks for opening the issue, if will be needed anything else - feel free to open new one

**Comment by foarsitter** [comment]
Thanks for you quick action!
